pkg/cmd/cli/describe/projectstatus.go

```diff
@@ -950,7 +952,11 @@ func describeDeployments(f formatter, dcNode *deploygraph.DeploymentConfigNode,
 	}
 
 	for i, deployment := range deploymentsToPrint {
-		out = append(out, describeDeploymentStatus(deployment.ReplicationController, i == 0, dcNode.DeploymentConfig.Spec.Test, restartCount))
+		if latestName == deployment.ReplicationController.Name {
+			out = append(out, describeDeploymentStatus(deployment.ReplicationController, i == 0, dcNode.DeploymentConfig.Spec.Test, restartCount))
+		} else {
+			out = append(out, describeDeploymentStatus(deployment.ReplicationController, i == 0, dcNode.DeploymentConfig.Spec.Test, 0))
```

Comment:
If this deployment is active ie. has pods but is about to be rolled out, I would want to know if its pods have restarted. I think we need the restart count to be part of the RC node.